src/db/oracledb/books-dao.js

```diff
@@ -15,9 +15,28 @@ import { getConnection } from './connection';
 const getBooks = async (query) => {
   const connection = await getConnection();
   try {
-    const parsedQuery = parseQuery(query); // contrib is changed
-    const { rawBooks } = await connection.execute(parsedQuery);
-    return rawBooks;
+    const parsedQuery = parseQuery(query);
+    const whereClause = generateWhereClause(parsedQuery).toLowerCase();
+
+    const selectQuery = `
+      SELECT *
+      FROM library_api_books
+      ${whereClause}
+    `;
+
+    delete parsedQuery['page[number]'];
+    delete parsedQuery['page[size]'];
```

Comment:
Why do we want to delete this? In addition, you are returning all books right now, but we should only return 25 per page by default: https://github.com/osu-mist/library-api/blob/main/openapi.yaml#L500

Reply:
It was for testing, and now removed these lines.

Reply:
25 per page implemented.